### PR TITLE
Feat : Add site_info to the title of the species accessed through Recordings or Unique species

### PR DESCRIPTION
**Pull request by alexbelgium**
Currently, the play.php page accessed through Recordings or Today species only shows as a text the common name of the specie

This PR takes advantage of the new site_info function to add a ebird/allaboutbirds link to the title, allowing a faster access and a bit more harmonized management with other pages

### CODE REVIEW

**Comment by alexbelgium** [comment]
Edit : sorry it messes up the display I need to find a way to get the sci_name from the title and not the spectrograms

**Comment by alexbelgium** [comment]
It works by adding a new common function (get_sci_name) which does the invert of (get_com_en_name) and then passes it to the (info_site) function. Although it adds a new common function, this seems in line with how the app is coded currently

**Comment by alexbelgium** [comment]
Reverted to draft, there is an issue in my get_sci_name function

**Comment by alexbelgium** [comment]
Working

**Comment by alexbelgium** [comment]
I'm thinking of extending this PR to harmonize visual & function between first page & recordings : 
- [ ] add mini graph to recordings
- [ ] propose an icon similar to mini graphs as an information to open the page - currently it is not obvious what open wikipedia ; species list ; or ebird/allaboutbirds

**Comment by Nachtzuster** [comment]
@alexbelgium it is hard to keep up with you :smile:
I'll have a look this WE!

**Comment by alexbelgium** [comment]
Thanks very much! This PR is more a prelude to

File: scripts/common.php [hunk 1]
```diff
@@ -103,6 +103,20 @@ function get_com_en_name($sci_name) {
   return $engname;
 }
 
+function get_sci_name($com_name) {
+  if (!isset($_labels_flickr)) {
+    $_labels_flickr = file(get_home() . "/BirdNET-Pi/model/labels.txt");
```
Comment (Nachtzuster) [inline scripts/common.php:108]: re-using `_labels_flickr` will break get_com_en_name() since one needs `labels_flickr.txt` and the other `labels.txt`

**Review by Nachtzuster — CHANGES_REQUESTED** [review]

File: scripts/common.php (hunk 1 above)
Comment (alexbelgium) [reply scripts/common.php:108]: Thanks ; bad copy paste

**Comment by alexbelgium** [comment]
Thanks for the review, corrected now

**Pull request merged**